File: JS/Mean of Array After Removing Some Elements.js
var trimMean = function(arr) {
    arr.sort((a,b) => a-b);
    let remove_position = (0.05)*arr.length;
    let start =remove_position;
    let end= arr.length-remove_position;
    let sum=0;
    let remaining_size = arr.length - (2*remove_position);
    for(let i=start ; i<end ; ++i ){
        sum += arr[i];
    }
    let mean = sum/remaining_size;
    return mean;
};

File: C++/Mean of Array After Removing Some Elements.cpp
class Solution {
public:
    double trimMean(vector<int>& arr) {
        sort(arr.begin(),arr.end());
        int remove_position = (0.05)*arr.size();
        int start =remove_position;
        int end= arr.size()-remove_position;
        double sum=0;
        int remaining_size = arr.size() - (2*remove_position);
        for(int i=start ; i<end ; ++i ){
            sum += arr[i];
        }
        double mean = sum/remaining_size;
        return mean;
    }
};